File: back/my-service/src/main/java/com/rcs/services/MyService.java
package com.rcs.services;

import com.rcs.models.MyModel;
import com.rcs.repos.MyRepository;
import org.springframework.stereotype.Service;

@Service
public class MyService {

    private final MyRepository myRepository;

    public MyService(MyRepository myRepository) {
        this.myRepository = myRepository;
    }

    public Iterable<MyModel> getMyModels() {
        return myRepository.findAll();
    }
}

File: back/my-service/src/main/kotlin/com/rcs/services/MyService.kt
package com.rcs.services

import com.rcs.models.MyModel
import com.rcs.repos.MyRepository
import org.springframework.stereotype.Service

@Service
class MyService(private val myRepository: MyRepository) {

    fun getMyModels(): Collection<MyModel> {
        return myRepository.findAll().toList()
    }
}
